app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/ViewerFragmentViewModel.java

```diff
@@ -0,0 +1,28 @@
+package com.teamagam.gimelgimel.app.model.ViewsModels;
+
+import com.teamagam.gimelgimel.domain.geometries.entities.PointGeometry;
+import com.teamagam.gimelgimel.domain.images.SendImageMessageInteractor;
+import com.teamagam.gimelgimel.presentation.presenters.SendImageMessagePresenter;
+
+import javax.inject.Inject;
+
+public class ViewerFragmentViewModel {
```

Comment:
we have other viewModel for the viewer fragment in the send message feature. you need also to bind the FAB to the method. for